src/main/java/unicash/logic/parser/HelpCommandParser.java

```diff
@@ -0,0 +1,22 @@
+package unicash.logic.parser;
+
+import unicash.logic.commands.HelpCommand;
+import unicash.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments for the list command.
+ */
+public class HelpCommandParser implements Parser<HelpCommand> {
+
+    /**
+     * Parses {@code userInput} into a command and returns it.
+     *
+     * @throws ParseException if {@code userInput} does not conform the expected format
+     */
+    @Override
+    public HelpCommand parse(String userInput) {
+        assert userInput != null : "userInput cannot be null";
```

Comment:
I think I can change this to requireNonNull, but I believe the key difference of assert is when you know that wherever is calling this method will strictly pass in non-null values and therefore you add another layer of check to assert it is not null. `requireNonNull` on the other hand is for cases where there could be other forms of input which could potentially lead to `null` as an input and you should show an appropriate response to deal with it.

But either ways, there shouldn't be much of a change in meaning since either ways `Parser` in our project is always called by `UniCashParser`.

Reply:
I see, no issues here, was just wondering